### PR TITLE
GT-545 - add Article Renderer to Short Cut

### PR DESCRIPTION
**Pull request by gyasistory**
Currently the questions about God shortcut deep links to the legacy activity, it should deep link to the new Articles related activities.

https://jira.cru.org/browse/GT-546

### CODE REVIEW

File: ui/shortcuts/build.gradle [hunk 1]
```diff
@@ -20,6 +20,7 @@ dependencies {
     implementation project(':ui:base-app')
     implementation project(':ui:everystudent')
```
Comment (frett) [inline ui/shortcuts/build.gradle]: this should be removed now

File: ui/shortcuts/src/main/java/org/cru/godtools/shortcuts/GodToolsShortcutManager.java [hunk 2]
```diff
@@ -61,7 +61,7 @@
 
 import static org.cru.godtools.base.Settings.PREF_PARALLEL_LANGUAGE;
 import static org.cru.godtools.base.Settings.PREF_PRIMARY_LANGUAGE;
-import static org.cru.godtools.model.Tool.CODE_EVERYSTUDENT;
+//import static org.cru.godtools.model.Tool.CODE_EVERYSTUDENT;
```
Comment (frett) [inline ui/shortcuts/src/main/java/org/cru/godtools/shortcuts/GodToolsShortcutManager.java]: go ahead and delete this import, don't leave it commented here :)

**Comment by gyasistory** [comment]
Ok I

**Comment by frett** [comment]
I'm going to rebase this to make sure it's up to date

**Comment by frett** [comment]
This will now download the Questions About God tool if you create a shortcut to it without adding the tool to your device

**Pull request merged**